src/components/AddRoomForm/AddRoomForm.tsx

```diff
@@ -0,0 +1,214 @@
+                    value={values.keyOwner}
+                    onChange={handleInputChange}
+                    name='keyOwner'
+                    select
+                    required
+                  >
+                    {users.map((selectableUser) => {
+                      return (
+                        <MenuItem
+                          key={selectableUser.id}
+                          value={selectableUser.name}
+                        >
+                          {selectableUser.name}
+                        </MenuItem>
+                      );
+                    })}
+                  </TextField>
+                </Grid>
+                <Grid
+                  item
+                  mobile={12}
+                >
+                  <Field
+                    as={TextField}
+                    className={styles.input}
+                    label={t('addRoom.roomNumber')}
+                    inputProps={{ inputMode: 'numeric', pattern: '[0-9]*' }}
+                    type='number'
+                    name='roomNumber'
+                    required
+                  />
+                </Grid>
+                <Grid
+                  item
+                  mobile={12}
+                >
+                  <Field
+                    as={TextField}
+                    className={styles.input}
+                    label={t('addRoom.floor')}
+                    inputProps={{ inputMode: 'numeric', pattern: '[0-9]*' }}
+                    type='number'
+                    name='floor'
+                    required
+                  />
+                </Grid>
+                <Grid
+                  item
+                  mobile={12}
+                >
+                  <Field
+                    as={TextField}
+                    className={styles.input}
+                    label={t('addRoom.maxCapacity')}
+                    inputProps={{ inputMode: 'numeric', pattern: '[0-9]*' }}
+                    type='number'
+                    name='maxCapacity'
+                    required
+                  />
+                </Grid>
+                <Grid
+                  item
+                  mobile={12}
+                >
+                  <Field
+                    as={TextField}
+                    InputLabelProps={{ shrink: true }}
+                    className={styles.input}
+                    label={t('addRoom.openingTime')}
+                    type='date'
+                    name='openingTime'
+                    required
+                  />
+                </Grid>
+                <Grid
+                  item
+                  mobile={12}
+                >
+                  <Field
+                    as={TextField}
+                    InputLabelProps={{ shrink: true }}
+                    className={styles.input}
+                    label={t('addRoom.closingTime')}
+                    type='date'
+                    name='closingTime'
+                    required
```

Comment:
This should be time no date
![Screenshot from 2023-04-25 13-54-09](https://user-images.githubusercontent.com/34687242/234155433-234e9588-87cc-4b8d-ba0a-c2ad5f94212e.png)